client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -97,7 +97,7 @@
         }, false);
 
         // send request
-        Galaxy.emit.debug('uploadbox::uploadpost()', 'Posting following data.', cnf);
+        Galaxy.logger.emit('debug', 'uploadbox::uploadpost()', [ 'Posting following data.', cnf ]);
```

Comment:
I think we should either keep or remove the emit helper function and not shift it out of Galaxy into Utils and then require Utils to access Galaxy.

I am not sure if the addLogging() function is an alternative. It has to be explicitly included and called in every class. This can be cumbersome, particularly for small classes with just one or two log statements. Additionally, addLogging() limits the number of namespaces per class to one i.e. _logNamespace.
In contrast, Galaxy.emit is directly available and designed like the common console. Instead of calling console.log(args), we can just call Galaxy.emit.log(namespace, args). This gives us the option to specify multiple and custom namespaces/tags for log statements e.g. the above tag 'uploadbox::uploadpost()' is more informative than just 'upload', particularly since we are currently unable to infer the file and line number.